### DISCUSSION TITLE
firefox wont access internet

### DISCUSSION THREAD

**Discussion by kelvinhub2** [jlesage/docker-firefox#208] [Q&A]
Hi
first of all, thank you to the community for this great piece of work!
I've installed the docker image on my synology and I can access the "landing" page of firefox as expected, including all the ff settings etc. However, **I can not access any website**. My container's network is in "bridge" mode. Does anyone know, why I can not access the internet?
Thank you in advance!

**Comment by jlesage** [comment]
Do you have the firewall enabled ?

You might have better answer by looking at the Synology forum.  For example, see https://community.synology.com/enu/forum/1/post/142221

**Reply by kelvinhub2** [reply]
Thank you so much. I don’t have the Synology firewall enabled, but will look for a solution more on the Synology side than on the docker image settings side
